shared/config.lua

```diff
@@ -8,6 +8,7 @@ Config.OnlyShowOnDuty = true
 -- YOU NEED TO SET THIS UP FOLLOW INSTRUCTIONS BELOW.
 -- Documents: https://docs.fivemerr.com/integrations/mdt-scripts/ps-mdt
 Config.FivemerrMugShot = true
+Config.MugshotEncoding = "png"  -- Default encoding is png, can be set to jpg for better performance.

```

Comment:
This comment is a bit odd. It sounds like people can set it to jpg, and then it will always perform better, if they don't know what encoding means.

Reply:
Additionally, this change is not backward compatible, as the default was previously 'jpg,' which many people depend on.

Reply:
Nooo, the default was png, I was just a little late uploading it as Monkey had already put in his own change for jpg :)

Reply:
We don't need a configuration for this. If people want to change the encoding format, they can just do it directly in the code instead.